frontend/src/features/permits/pages/Application/components/form/VehicleDetails/VehicleDetails.tsx

```diff
@@ -175,47 +176,57 @@ export const VehicleDetails = ({
 
   // Find vehicle subtypes that are allowed by LOAs
   const permittedLOAPowerUnitIds = new Set([
-    ...selectedLOAs.map(loa => loa.powerUnits)
```

Comment:
I have made a lot of changes in the VehicleDetails component, which included removing some of these variables... You might need to wait until that PR is merged, and then we can consolidate your changes along with my changes.

Reply:
I didn't actually touch this code, I think my prettier must have formatted this file differently on save hence the changes here. Please disregard